scout/server/blueprints/cases/views.py

```diff
@@ -439,7 +387,14 @@ def phenotypes_actions(institute_id, case_name):
             store.remove_phenotype(institute_obj, case_obj, user_obj, case_url, hpo_id)
 
     if action == "ADDGENE":
-        hgnc_ids = parse_raw_gene_ids(request.form.getlist("genes"))
+        try:
+            hgnc_ids = parse_raw_gene_ids(request.form.getlist("genes"))
+        except ValueError:
+            flash(
+                "Provided gene info could not be parsed! " "Please allow autocompletion to finish.",
```

Comment:
In which case does this crash? If I don't type something that it likes I just get this validation error:

<img width="552" alt="image" src="https://user-images.githubusercontent.com/28093618/185885582-3f077bb5-28b2-4dc2-a2ea-bb4fe0b2c10c.png">

Reply:
It's possible, I've managed. But don't remember how. 

Anyways, the function `parse_raw_gene_ids()` was extracted from cases/views. Here it had the side-effect of flashing a warning. When moving the functionality to utils/genes.py the side-effect of flashing messages had to be extracted. Therefor to keep the same behaviour the try/except now is outside and kept the same. cases.views.py and panels.views.py will be similar.